packages/core/src/browser/storage-service.ts

```diff
@@ -18,6 +18,7 @@ import { inject, injectable, postConstruct } from 'inversify';
 import { ILogger } from '../common/logger';
 import { MessageService } from '../common/message-service';
 import { WindowService } from './window/window-service';
+const isElectron: () => boolean = require('is-electron');
```

Comment:
I don't believe we want to add an explicit dependency to `electron` from `core` anymore. https://github.com/eclipse-theia/theia/pull/4873

Reply:
Yes, I forgot to test the browser version.  I've changed it to use ElectronEnv and tested it on both browser and electron.  Thanks for pointing me to the PR with the discussion.